core/src/main/kotlin/com/linecorp/kotlinjdsl/query/clause/function/QueryFunctionClause.kt

```diff
@@ -0,0 +1,9 @@
+package com.linecorp.kotlinjdsl.query.clause.function
+
+import com.linecorp.kotlinjdsl.query.spec.Froms
+import javax.persistence.criteria.AbstractQuery
+import javax.persistence.criteria.CriteriaBuilder
+
+interface QueryFunctionClause {
+    fun apply(froms: Froms, query: AbstractQuery<*>, criteriaBuilder: CriteriaBuilder)
+}
```

Comment:
-- korean
QueryFunctionClause는 어디서 사용되는 것인가요?

-- english
Where is the Query FunctionClause used?

Reply:
-- korean
Clause 클래스는 Where 이나 OrderBy와 같은 절을 구현하기 위해 있는 것이기 때문에, Function을 위해 Clause 클래스를 구현할 필요는 없을 것 같습니다.

-- english
It's not necessary to implement `Clause class` for Function, because the `Clause class` must be implemented for clauses such as `Where` and `OrderBy`.

Reply:
-- korean 
맞습니다. 처음에 개념을 착각해서 추가해놓고 실수로 삭제를 안했네요. 삭제하도록 하겠습니다.

-- english
Yes. At first, I misunderstood the concept, so I added it and didn't delete it by mistake. I will delete it.

Reply:
-- korean
삭제 되었습니다.

-- english
deleted.